src/components/DaySlot.jsx

```diff
@@ -0,0 +1,225 @@
+import React, { PropTypes, Component } from 'react'
+import { findDOMNode } from 'react-dom'
+import cn from 'classnames';
+
+import SelectableTimeGutter from '../containers/SelectableTimeGutter.jsx'
+import EventRow from '../EventRow.jsx'
+
+import dates from '../utils/dates';
+import { isSelected } from '../utils/selection';
+import { notify } from '../utils/helpers';
+import { accessor, elementType } from '../utils/propTypes';
+import { accessor as get } from '../utils/accessors';
+import localizer from '../localizer'
+
+export default class DaySlot extends Component {
+  static propTypes = {
+    formatter: PropTypes.func,
```

Comment:
no, this is the code that displays the selection div.  It's now external to the component to demonstrate how simple and easy it is to expose this in a controlled way so that users can extend it in unique ways